meson/dependencies/meteoIO/meson.build

```diff
@@ -0,0 +1,50 @@
+_opt = 'METEOIO'
+_dir = get_option(_opt+'_DIR')
+_libname = _opt.to_lower()
+
+_code = '''
+#include <meteoio/Config.h>
```

Comment:
Why do need to include this file? It should be already included in __MeteoIO.h__.
I've tried to remove this line and I was able to compile..

Reply:
this is just a toy code to test if I can actually find a header of the meteoio library. which file is not important. Basically, I am testing if the `METEOIO_DIR` is correct. The code written there has nothing to do with geotop